pkg/reconciler/revision/table_test.go

```diff
@@ -661,6 +663,15 @@ func withK8sServiceName(sn string) RevisionOption {
 	}
 }
 
+func withDefaultContainerStatuses(name string) RevisionOption {
+	return func(r *v1.Revision) {
+		r.Status.ContainerStatuses = []v1.ContainerStatuses{{
+			Name:        name,
+			ImageDigest: "",
```

Comment:
I don't see any variation in the names being passed here so I think we can simplify this

```suggestion
func withDefaultContainerStatuses() RevisionOption {
	return func(r *v1.Revision) {
		r.Status.ContainerStatuses = []v1.ContainerStatuses{{
			Name:        rev.Name,
			ImageDigest: "",
```